packages/dev/core/src/DeviceInput/eventFactory.ts

```diff
@@ -76,6 +76,15 @@ export class DeviceEventFactory {
             evt.pointerType = "touch";
         }
 
+        let buttons = 0;
+
+        // Populate buttons property with current state of all mouse buttons
+        // Uses values found on: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons
+        buttons += deviceInputSystem.pollInput(deviceType, deviceSlot, PointerInput.LeftClick);

```

Comment:
just making sure - pollInput returns 0 or 1?

Reply:
Yes, that is correct.